backend/services/workflows/client/nats/client.go

```diff
@@ -16,26 +16,28 @@ package nats
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"time"
 
 	natsio "github.com/nats-io/nats.go"
+	"github.com/pkg/errors"
 
 	"github.com/mendersoftware/mender-server/pkg/log"
 )
 
+type ConsumerMode int
+
 const (
+	PushMode = ConsumerMode(iota)
+	PullMode
+
+	defaultConsumerMode = PullMode
```

Comment:
I don't understand the value of introducing this const which is used once. I believe it would be more clear to simply use `PullMode` wherever this is used to avoid the extra indirection.